modules/installation-requirements-bare-metal.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-requirements-bare-metal-{context}"]
+= Bare metal machine requirements
+
+With a bare metal installation, you must deploy all of the required machines.
+
+[id="machine-requirements-{context}"]
+== Required machines
+
+The smallest {product-title} clusters require the following hosts:
+
+* 1 bootstrap machine
+
+* 3 control plane machines
+
+* At least 1 worker machine
+
+[NOTE]
+====
+The cluster requires the bootstrap machine to deploy the {product-title} cluster
+on the 3 control plane machines. You can remove the bootstrap machine after you
+install the cluster.
+====
+
+The bootstrap and control plane machines must use {op-system-first} as the operating system.
+
+//Need to provide details about the minimum resource requirements
+////
+[id="minimum-resource-requirements-{context}"]
```

Comment:
@abhinavdahiya, when might these minimum requirements be available, and do you expect someone from the install team to provide them?

Reply:
:eyes: looking

Reply:
@abhinavdahiya, will you PTAL at the information that I extrapolated from the smallest AWS instances? Do you know what the minimum storage requirements are?

Reply:
Abhinav sent the requirements in Slack, and I've updated them.